addons/mrp_subcontracting/views/subcontracting_portal_templates.xml

```diff
@@ -54,15 +58,19 @@
         </t>
     </template>
 
-    <template id="subcontracting_portal" name="Subcontracting View in Portal">
-        <t t-call="portal.frontend_layout">
-            <t t-set="no_footer" t-value="true"/>
-            <t t-call="mrp_subcontracting.subcontracting"/>
+    <template id="subcontracting_portal" name="Subcontracting View in Portal" inherit_id="portal.portal_sidebar" primary="True">
+    <t t-call="portal.portal_layout">
+        <t t-set="breadcrumbs_searchbar" t-value="True"/>
+        <t t-call="portal.portal_searchbar">
+            <t t-set="title" >Productions</t>
         </t>
+        <t t-set="no_footer" t-value="true"/>
+        <t t-call="mrp_subcontracting.subcontracting"/>
+    </t>
     </template>
 
     <template id="subcontracting" name="Subcontracting Portal View">
-        <iframe width="100%" height="100%" frameborder="0" t-attf-src="/my/productions/{{ picking_id }}/subcontracting_portal"/>
+        <iframe width="100%" height="700px" frameborder="0" t-attf-src="/my/productions/{{ picking_id }}/subcontracting_portal"/>
```

Comment:
Not sure that it's the correct solution, but setting the height to ``100%`` even in the parent container does not seem to work

Reply:
hmm I am not a 100% sure, but I think it should work if you add the `iframe` in a `div`